packages/components/psammead-radio-schedule/src/testHelpers/helper.jsx

```diff
@@ -41,17 +42,19 @@ export const renderProgramCard = ({
   duration = 'PT30M',
   durationLabel = 'Duration',
   startTime = 1566914061212,
+  displaySummary = boolean('show summary', true),
```

Comment:
Added a boolean knob to show/hide program card summary - though a bit unconventionally, as trying to add the knob to `psammead-radio-schedule/src/index.stories.jsx` led to it not working consistently/at all